game.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en" class="vh-100">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<script src="https://cdn.tailwindcss.com"></script>
+        <script>
+            tailwind.config = {
```

Comment:
looks good, we should prob move this to the shared js file ito keep our breakpoints consistent

Reply:
looks good to me too. But seems like we've use tailwind which has not been approved.